rosetta/Dockerfile

```diff
@@ -5,7 +5,7 @@ COPY . .
 # Proxy
 WORKDIR /elrond/cmd/proxy
 RUN go build
-# $(ls /go/pkg/mod/github.com/!elrond!network/ | grep arwen-wasm-vm | sed 's/.* //' | head -1) this command is required  to be able to exact the full name of folder that starting with 'arwen-wasm-wm'
+# $(ls /go/pkg/mod/github.com/!elrond!network/ | grep arwen-wasm-vm | sed 's/.* //' | head -1) this command is required  to be able to extract the full name of folder that starting with 'arwen-wasm-wm'
```

Comment:
`the full name of folder that starting with 'arwen-wasm-wm'`  =>  `the full name of the directory that starts with 'arwen-wasm-wm'`